### PR TITLE
[FIX] Refactoring hard-coded value into value Initialization 

### PR DESCRIPTION
**Pull request by jananjoy**

### CODE REVIEW

File: project_event/models/project_task.py [hunk 1]
```diff
@@ -6,9 +6,26 @@
 from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError
 
+ZERO = 0
+ONE = 1
+EMPTY = ''
+MINUTES = 60
```
Comment (rimbendhaou) [inline project_event/models/project_task.py]: if we will follow your logic, it should be MINUTES_IN_HOUR like hour_in_day

File: project_event/models/project_task.py [hunk 2]
```diff
@@ -627,7 +644,7 @@ def _compute_order_task(self):
     def get_task_order(task_ds, activity_ds, format):
         time_diff = datetime.strptime(task_ds, format) \
                     - datetime.strptime(activity_ds, format)
-        return time_diff.days * 24 * 60 + time_diff.seconds / 60
+        return time_diff.days * HOUR_IN_DAY * MINUTES + time_diff.seconds / 60
```
Comment (rimbendhaou) [inline project_event/models/project_task.py]: what about the /60

Comment (rimbendhaou) [reply project_event/models/project_task.py]: in this case the second 60 should be SECONDS_IN_MINUTE

File: project_event/models/project_task.py (hunk 1 above)
Comment (rimbendhaou) [reply project_event/models/project_task.py]: and the second one SECONDS_IN_MINUTE = 60

File: project_event/models/project_task.py [hunk 3]
```diff
@@ -871,16 +890,13 @@ def child_reservation(child):
     def get_message_body(action):
         switcher = {
             'draft': ' ',
-            'option': _('The following is optional and \
-                        appears as crosshatched on your calendar'),
-            'requested': _('The following is requested'),
-            'accepted': _('The following is approved'),
+            'option': _(MSG_OPTION),
```
Comment (eilst) [inline project_event/models/project_task.py:903]: Did you check this works for translations? @jananjoy

File: project_event/models/project_task.py [hunk 4]
```diff
@@ -6,9 +6,27 @@
 from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError
 
+ZERO = 0
```
Comment (eilst) [inline project_event/models/project_task.py]: no use zero=0 could this be different? @jananjoy

File: project_event/models/project_task.py [hunk 5]
```diff
@@ -6,9 +6,27 @@
...
+ONE = 1
```
Comment (eilst) [inline project_event/models/project_task.py]: same

File: project_event/models/project_task.py [hunk 6]
```diff
@@ -6,9 +6,27 @@
...
+EMPTY = ''
```
Comment (eilst) [inline project_event/models/project_task.py]: same

File: project_event/models/project_task.py [hunk 7]
```diff
@@ -6,9 +6,27 @@
...
+MINUTES_IN_HOUR = 60
```
Comment (eilst) [inline project_event/models/project_task.py:9]: this is good, it makes sense. it's more explicit

File: project_event/models/project_task.py (hunk 3 above)
Comment (jananjoy) [reply project_event/models/project_task.py:903]: @rimbendhaou  it works

**Review by eilst — APPROVED** [review]

**Pull request merged**